tests/integration/test_commands.py

```diff
@@ -1,20 +1,1289 @@
+
+        # command has no (python) errors, has exit code 0
+        assert result.exit_code == 0
+        # command output has file creation messages for 'Patient' and 'Biospecimen' manifests
+        assert result.output.split("\n")[7] == (
+            "Find the manifest template using this CSV file path: "
+            "tests/data/example.Biospecimen.manifest.csv"
+        )
+        assert result.output.split("\n")[10] == (
+            "Find the manifest template using this CSV file path: "
+            "tests/data/example.Patient.manifest.csv"
+        )
+
+        # manifests have expected columns
+        assert list(biospecimen_df.columns) == [
+            "Sample ID",
+            "Patient ID",
+            "Tissue Status",
+            "Component",
+        ]
+        assert list(patient_df.columns) == [
+            "Patient ID",
+            "Sex",
+            "Year of Birth",
+            "Diagnosis",
+            "Component",
+            "Cancer Type",
+            "Family History",
+        ]
+        # manifests only have one row
+        assert len(biospecimen_df.index) == 1
+        assert len(patient_df.index) == 1
+        # manifests are empty except for component column which contains the name of the component
+        assert biospecimen_df["Component"].to_list() == ["Biospecimen"]
+        assert patient_df["Component"].to_list() == ["Patient"]
+        for column in ["Sample ID", "Patient ID", "Tissue Status"]:
+            assert np.isnan(biospecimen_df[column].to_list()[0])
+        for column in [
+            "Patient ID",
+            "Sex",
+            "Year of Birth",
+            "Diagnosis",
+            "Cancer Type",
+            "Family History",
+        ]:
+            assert np.isnan(patient_df[column].to_list()[0])
+
+    def test_generate_empty_google_sheet_manifests(
+        self,
+        runner: CliRunner,
+        testing_config: ConfigurationForTesting,
+    ) -> None:
+        """
+        Tests for:
+        - command has no errors, has exit code 0
+        - command output has file creation messages for 'Patient' and 'Biospecimen' manifest csvs
+        - command output has file creation messages for 'Patient' and 'Biospecimen' manifest links
+
+        both google sheets:
+        - have drop downs are populated correctly
+        - required fields are marked as “light blue”,
+        - non-required field are marked as white.
+        - first row comments are 'TBD'
+
+        Patient google sheet:
+        - first row of 'Family History has its own comment
+
+        Manual tests:
+        - Open 'CLI_TestManifestCommand_google_sheet_empty_patient.xlsx'
+        - Select 'Diagnosis' to be 'cancer' in the first row:
+            - 'Cancer Type' and 'Family History' cells in first row should be light blue.
+        - Select 'Diagnosis' to NOT be 'cancer' in the first row:
+            - 'Cancer Type' and 'Family History' cells in first row should be white
```

Comment:
For any tests that need manual verification please use the pytest mark `manual_verification_required`

https://github.com/Sage-Bionetworks/schematic/blob/778bf54db9c5b4de0af334c4efe034b3dde0b348/pytest.ini#L17

Reply:
Will do